neural_compressor/adaptor/ox_utils/smooth_quant.py

```diff
@@ -330,6 +336,37 @@ def conv(node, scale):  # pragma: no cover
                 )
             return True
 
+        def mul_add(node, scale):  # pragma: no cover
+            node_parent = self.model.get_parent(node, 0)
+            if not len(self.model.get_parents(node)) == 1 or node_parent.op_type != "Mul":

```

Comment:
below comment said "Add has itself a MatMul before" but here is "node_parent.op_type != 'Mul'", could you align it?